interfaces/Portalicious/eslint.config.js

```diff
@@ -89,21 +92,22 @@ module.exports = tseslint.config(
         {
           checkId: false,
           ignoreAttributes: [
+            'app-query-table[localStorageKey]',
             'data-testid',
-            'icon',
-            'iconPos',
-            'ngSrc',
-            'styleClass',
-            'position',
-            'pSortableColumn',
             'field',
-            'display',
-            'sortField',
-            'appendTo',
-            'iconPosition',
-            'stateStorage',
-            'stateKey',
-            'localStorageKey',
+            'img[ngSrc]',
+            'inputStyleClass',
+            'ng-container[slot]',
+            'p-button[icon]',
+            'p-button[iconPos]',
+            'p-columnFilter[display]',
+            'p-contextMenu[appendTo]',
+            'p-iconField[iconPosition]',
+            'p-sidebar[position]',
+            'p-table[stateKey]',
+            'p-table[stateStorage]',
+            'styleClass',
+            'th[pSortableColumn]',
```

Comment:
Was this auto-generated somehow? If not, I fear that the specificity on some of these might come back to bite us (for example, I can imagine `icon` being relevant for some other component too)

Reply:
It was auto-sorted alphabetically. ;)

The condition I used to add a more specific element to the selector was their current use-cases in the code-base.
So, in the future, if there are other elements that really need the exception, the selector can/should be amended.

The easiest automated solution would be to use a list of "supported attributes per component" provided by any library we use (i.e. PrimeNG etc.); But that doesn't excist, so we could build it up over time...
Or use another reminder to "add attributes to support translations"...